roles/fzf/meta/main.yml

```diff
@@ -0,0 +1,20 @@
+galaxy_info:
+  author: Tim Hourigan
+  description: Install fzf
+  license: MIT
+  min_ansible_version: 2.5
+  role_name: fzf
+  namespace: timhourigan
+  platforms:
```

Comment:
From a first pass of the code it looks like this also supports Fedora. Is it worth adding it here?

Reply:
See below :smile: 

I'm going to try to support Debian, Fedora (34 for now) and Ubuntu.

Reply:
You can see from the molecule file, that all 3 flavours are being tested.